server/routes/serviceProviderReferrals/dashboardPresenter.ts

```diff
@@ -51,26 +67,28 @@ export default class DashboardPresenter {
       { text: referralSummary.referenceNumber, sortValue: null, href: null },
       {
         text: utils.convertToTitleCase(
-          `${referralSummary.serviceUserFirstName ?? ''} ${referralSummary.serviceUserLastName ?? ''}`
+          `${referralSummary.referral.serviceUser.firstName ?? ''} ${
+            referralSummary.referral.serviceUser.lastName ?? ''
+          }`
         ),
-        sortValue: `${referralSummary.serviceUserLastName ?? ''}, ${
-          referralSummary.serviceUserFirstName ?? ''
+        sortValue: `${referralSummary.referral.serviceUser.lastName ?? ''}, ${
+          referralSummary.referral.serviceUser.firstName ?? ''
         }`.toLocaleLowerCase('en-GB'),
         href: null,
       },
-      { text: referralSummary.interventionTitle, sortValue: null, href: null },
+      { text: interventionForReferral.title, sortValue: null, href: null },
       this.showAssignedCaseworkerColumn
         ? null
-        : { text: referralSummary.assignedToUserName ?? '', sortValue: null, href: null },
+        : { text: referralSummary.assignedTo?.username ?? '', sortValue: null, href: null },
```

Comment:
unrelated to these changes, but in the PP dashboard i think we show 'Unassigned' for these types of cases - do we know why this is different?

Reply:
I do not have an answer; but we could push this into config to make it look like an explicit choice -- `dashboards.serviceProvider.unassignedLabel` 🤷